.github/workflows/pre-main.yaml

```diff
@@ -4,7 +4,7 @@ on:
   push:
     branches: [ main ]
   pull_request:
-    branches: [ main ]
+    branches: [ main, ginkgo_removal]
```

Comment:
The references to the `ginkgo_removal` branch should be deleted.

Reply:
We'll have to do that in a subsequent PR as well.  Same thing in the QE PR.